packages/yoga/src/helpers.ts

```diff
@@ -47,3 +50,142 @@ export function invalidateImportCache(files: string[]) {
     }
   })
 }
+
+/**
+ * Find a config file path
+ */
+export function findConfigFile(fileName: string) {
+  return ts.findConfigFile(
+    /*searchPath*/ process.cwd(),
+    ts.sys.fileExists,
+    fileName,
+  )
+}
+
+/**
+ * - Compute paths relative to the root of the project
+ * - Set defaults if some options are missing
+ */
+function normalizeConfig(rootPath: string, config: InputConfig): Config {
+  if (config.resolversPath) {
+    config.resolversPath = relativeToRootPath(rootPath, config.resolversPath)
+  } else {
+    config.resolversPath = relativeToRootPath(rootPath, './src/graphql')
+  }
+
+  // Context path is optional and should remain undefined if none is provided or if the default path doesn't exist
+  if (config.contextPath) {
+    config.contextPath = relativeToRootPath(rootPath, config.contextPath)
+  } else {
+    const contextPath = relativeToRootPath(rootPath, './src/context.ts')
+    if (fs.existsSync(contextPath)) {
+      config.contextPath = contextPath
+    }
+  }
+
+  if (!config.output) {
+    config.output = {}
+  }
+
+  if (config.output.typegenPath) {
+    config.output.typegenPath = relativeToRootPath(
+      rootPath,
+      config.output.typegenPath,
+    )
+  } else {
+    config.output.typegenPath = relativeToRootPath(
+      rootPath,
+      './src/generated/nexus.ts',
```

Comment:
[Optional] [Idea] There seems to be a lots of logic relying on existence of some files in `generated` folder. It will be nice to have some internal variable or even config that will have by default `./src/generated`. While nexus is going to use this folder there may be a case that people would like to customize this after generation. By having this simple option developers could adjust folder for their needs etc. No strong opinion about it so feel free to ignore it.